index.js

```diff
@@ -110,6 +113,15 @@ module.exports = {
   validateDependencies() {
     let bowerDependencies = this.app.project.bowerDependencies();
 
+    if (this.getBootstrapVersion() === 4) {
+      let checker = new VersionChecker(this);
+      let dep = checker.for('bootstrap');
+
+      if (!dep.gte(minimumBS4Version)) {
```

Comment:
How well does it handle comparisons between versions with "alpha" and "beta" tags?

Reply:
Yeah, I was also unsure, so I checked this using the underlying `semver` package. And it works as we need it, for alpha `gte` will return false! (seems to compare alphabetically I assume)